src/cephadm/cephadm

```diff
@@ -2234,7 +2234,8 @@ def get_container_mounts(ctx, fsid, daemon_type, daemon_id,
     if daemon_type == 'osd':
         mounts['/sys'] = '/sys'  # for numa.cc, pick_address, cgroups, ...
         # selinux-policy in the container may not match the host.
-        mounts['/usr/share/empty'] = '/sys/fs/selinux:ro'
+        if os.path.exists('/usr/share/empty'):
+            mounts['/usr/share/empty'] = '/sys/fs/selinux:ro'
```

Comment:
This won't fix the issue on OSes that do not have the empty dir. Can we create it instead?
If we wanted to be super-safe we could create a tmp dir in `/tmp` (something like `mktemp -d`) and use that one for mounting.

Reply:
I wish we had this (quite important) discussion on the initial PR.

Reply:
> This won't fix the issue on OSes that do not have the empty dir. Can we create it instead?
> If we wanted to be super-safe we could create a tmp dir in `/tmp` (something like `mktemp -d`) and use that one for mounting.

I'm ok with both. `/tmp` is not going to work, as we're using this function to generate the container mounts for the deployed containers as well.  maybe an empty folder within the unit's directory?

Reply:
I have used the cluster folder for the empty folder. The cluster folder will always exist.

Reply:
Yeah, better to have this on `/var/` instead of `/usr` (CoreOS mounts `/usr` read-only IIRC))